app/controllers/users_controller.rb

```diff
@@ -30,10 +30,11 @@ def create
     set_user_fields
 
     if @user.errors.empty? && @user.save
+      send_email_confirmation_mail(@user)
       if @bare_layout
-        render text: '<div class="success" style="font-size: 14pt; margin: 10pt;">User account created.</div>', layout: true
+        render text: '<div class="success" style="font-size: 14pt; margin: 10pt;">User account created. Please confirm your email address to continue.</div>', layout: true
       else
-        flash[:notice] = 'User account created. You can now log in.'
+        flash[:notice] = 'User account created. Please confirm your email address to continue.'
```

Comment:
This could be bit more clear for user, clearly asking to check your emails. Bit like it was in another notice, which was like "Check your emails and click the confirmation link..."